ddtrace.gemspec

```diff
@@ -28,6 +29,8 @@ Gem::Specification.new do |spec|
     raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
   end
 
+  # rubocop:disable all
+  # DEV: `spec.files` is more problematic than a simple Rubocop pass.
```

Comment:
"RuboCop Packaging" has a lot of opinions on this line: it will be mostly rewritten to comply. So I left it untouched for now.